functional_test/functional_test.go

```diff
@@ -27,6 +27,10 @@ var TESTENV = struct {
 	DefaultDomain        string `env:"TESTENV_DEFAULT_DOMAIN" envDefault:"example.com"`
 }{}
 
+func GenerateScopeID() string {
```

Comment:
btw, no need to make this function public, especially in the `_test.go` files, `generateScopeID` should work as well

Reply:
😅